src/vaadin-text-field-mixin.html

```diff
@@ -480,7 +480,7 @@
      */
     _valueChanged(newVal, oldVal) {
       // setting initial value to empty string, skip validation
-      if (newVal === '' && oldVal === undefined) {
+      if (oldVal === undefined) {
```

Comment:
The visual tests are failing for a good reason. This change skips not only validation but also value property propagation to the input element. Only validation should be skipped.